community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/sampling/IndexSamplingJobTracker.java

```diff
@@ -49,6 +51,7 @@ public boolean canExecuteMoreSamplingJobs()
         lock.lock();
         try
         {
+            assertNotStopped();
```

Comment:
Can we instead of throwing an exception just return false in `canExecuteMoreSamplingJobs` in case if it was already topped?